projects/plugins/jetpack/modules/publicize/publicize.php

```diff
@@ -946,45 +1069,45 @@ function register_post_meta() {
 	 *
 	 * Attached to the `save_post` action.
 	 *
-	 * @param int $post_id
-	 * @param WP_Post $post
-	 * @return void
+	 * @param int     $post_id Post ID.
+	 * @param WP_Post $post Post object.
 	 */
-	function save_meta( $post_id, $post ) {
-		$cron_user = null;
+	public function save_meta( $post_id, $post ) {
+		$cron_user   = null;
 		$submit_post = true;
 
-		if ( ! $this->post_type_is_publicizeable( $post->post_type ) )
+		if ( ! $this->post_type_is_publicizeable( $post->post_type ) ) {
 			return;
+		}
 
-		// Don't Publicize during certain contexts:
+		// Don't Publicize during certain contexts.
 
 		// - import
-		if ( defined( 'WP_IMPORTING' ) && WP_IMPORTING  ) {
+		if ( defined( 'WP_IMPORTING' ) && WP_IMPORTING ) {
 			$submit_post = false;
 		}
 
-		// - on quick edit, autosave, etc but do fire on p2, quickpress, and instapost ajax
+		// - on quick edit, autosave, etc but do fire on p2, quickpress, and instapost ajax.
 		if (
 			defined( 'DOING_AJAX' )
 		&&
 			DOING_AJAX
 		&&
-			!did_action( 'p2_ajax' )
+			! did_action( 'p2_ajax' )
 		&&
-			!did_action( 'wp_ajax_json_quickpress_post' )
+			! did_action( 'wp_ajax_json_quickpress_post' )
 		&&
-			!did_action( 'wp_ajax_instapost_publish' )
+			! did_action( 'wp_ajax_instapost_publish' )
 		&&
-			!did_action( 'wp_ajax_post_reblog' )
+			! did_action( 'wp_ajax_post_reblog' )
 		&&
-			!did_action( 'wp_ajax_press-this-save-post' )
+			! did_action( 'wp_ajax_press-this-save-post' )
 		) {
 			$submit_post = false;
 		}
 
 		// - bulk edit
-		if ( isset( $_GET['bulk_edit'] ) ) {
+		if ( filter_input( INPUT_GET, 'bulk_edit', FILTER_VALIDATE_BOOL ) ) {
```

Comment:
I think we often just do a check against `true` or `false` to validate booleans

Reply:
Judging by other instances of this check, it might be better to just leave the `isset` and use `// phpcs:ignore WordPress.Security.NonceVerification.Recommended`.

Reply:
If that's what we're doing elsewhere, then it makes sense. Looking at this some more, I wonder if it should be `! empty` rather than `isset`. It's unlikely that `bulk_edit` would be set, but a false value, but I suppose it's possible.

Reply:
Changed in 76bcac4